docs/api_reference/c_api/index.md

```diff
@@ -0,0 +1,15 @@
+# Zarr Streaming C API Reference
+
+Information on the `acquire_zarr` C implementation and how to use it. Check
+https://github.com/acquire-project/acquire-zarr for more information.
+
+<div class="cards">
+    <div class="card">
+        <h4><pre>acquire.zarr.h</pre></h4>
```

Comment:
Can we say something like
```suggestion
        <h4>The primary header file that provides the main streaming functionality and settings configuration.</h4>
```
?

Reply:
Would you like this to be the tile title, or the contents? We could have something like

```suggestion
        <h4><pre>acquire.zarr.h</pre></h4>
        <p>The primary header file that provides the main streaming functionality and settings configuration.</p>
```

Wdyt?

Reply:
Like this: 
![Captura de imagem_20250218_173320](https://github.com/user-attachments/assets/837ce567-1960-4263-a93c-bb921e631e81)

Reply:
Oh yeah, that's much better.